prep-html-links/index.html

```diff
@@ -5,6 +5,6 @@
 </head>
 <body>
   <h1>Home page</h1>
-  <a href="contact-us/contact-us.html"> Click here to go to the contact page </a>
+  <a href="contact-us/index.html"> Click here to go to the contact page </a>
```

Comment:
Unfortunately, this link won't work because there is no `index.html` in the `contact-us` directory.